package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "description": "package.json intended for in-repo use only, package.json used for publishing is located in src/AutoRest.CSharp/package.json",
   "devDependencies": {
-    "@azure-tools/cadl-csharp": "src/CADL.Extension/Emitter.Csharp",
+    "@azure-tools/typespec-csharp": "src/CADL.Extension/Emitter.Csharp",
     "@azure-tools/cadl-ranch-mockapis": "file:test/CadlRanchMockApis",
```

Comment:
Could you rename cadl-ranch-mockapis as well?

Reply:
cadl-ranch have not updated the name. For consistent, we will delay to update this until cadl-ranch updates the name.  And will update all the test project folder name to remove `cadl`, such as `cadlRanchProjects`, `cadlRanchMockApis` and so on.